File: demo/python/workers.py
from WorkerGroup import WorkerGroup
from WorkerA import WorkerA
from WorkerB import WorkerB
from WorkerC import WorkerC

# Running multiple workers
WorkerGroup([WorkerA(), WorkerB(), WorkerC()])

# Running a single worker
# worker = WorkerA()
# worker.start_work()

File: demo/js/workers.ts
// eslint-disable-next-line @typescript-eslint/no-var-requires
require('dotenv').config()

import WorkerGroup from '../../src/WorkerGroup'

import WorkerA from './WorkerA'
import WorkerB from './WorkerB'
import WorkerC from './WorkerC'

new WorkerGroup([
  new WorkerA(),
  new WorkerB(),
  new WorkerC()
])
